apps/voting_web/.gitignore

```diff
@@ -0,0 +1,28 @@
+# App artifacts
```

Comment:
I kinda like each "app" having it's own .gitignore since some things here don't need to be ignored in the other "apps". But yeah they could all be moved up to the root without a lot of fuss.